lua/telescope/_extensions/project.lua

```diff
@@ -0,0 +1,85 @@
+local has_telescope, telescope = pcall(require, 'telescope')
+
+if not has_telescope then
+	error('This plugins requires nvim-telescope/telescope.nvim')
+end
+
+local actions = require("telescope.actions")
+local finders = require("telescope.finders")
+local pickers = require("telescope.pickers")
+local conf = require("telescope.config").values
+
+local project_actions = require("project_actions")
+
+local project_dirs_file = vim.fn.stdpath('data') .. '/telescope-projects.txt'
+
+require('telescope').setup {
+		defaults = {
+			mappings = {
+				n = {
+					['d'] = project_actions.delete_project,
+					['c'] = project_actions.add_project,
+					['f'] = project_actions.find_project_files,
+				}
+			}
+		}
+	}
```

Comment:
That would add mapping right into the user config for all pickers builtin and extensions and custom. I don't even know if it is possible to run setup twice. Best case we are just extending worst case we are overriding all user mappings

Reply:
@Conni2461 eee okay! Will adjust now, thanks

Reply:
`register_extension` would give you a `setup` function with which something like that should be possible i think but i would still suggest checking out other solutions. Especially for `delete_project` and `find_project_files`.
https://github.com/nvim-telescope/telescope-fzy-native.nvim/blob/62bd7dc48ea9caf8c2b847b3d6e246a02efb90f7/lua/telescope/_extensions/fzy_native.lua#L13-L41 should give you some idea how to use it.